### PR TITLE
RPN-513: get creator from opensea lazymint

### PR DESCRIPTION
**Pull request by mystdeim**

### CODE REVIEW

File: nft/api/src/main/resources/config/application.yml [hunk 1]
```diff
@@ -58,4 +58,5 @@ api:
     media-meta-timeout: 10000
 
 common:
-  cryptoPunksContractAddress: "0xb47e3cd837dDF8e4c57F05d70Ab865de6e193BBB"
\ No newline at end of file
+  cryptoPunksContractAddress: "0xb47e3cd837dDF8e4c57F05d70Ab865de6e193BBB"
+  openseaLazyMintAddress: "0x495f947276749ce646f68ac8c248420045cb7b5e"
```
Comment (evgenynacu) [inline nft/api/src/main/resources/config/application.yml:62]: this should go in env specific config I think

File: nft/core/src/main/kotlin/com/rarible/protocol/nft/core/service/item/ItemCreatorService.kt [hunk 2]
```diff
@@ -59,8 +64,14 @@ class ItemCreatorService(
             }
     }
 
+    private fun fetchOpenseaCreator(tokenId: EthUInt256, itemId: ItemId): Mono<Address> {
+        logger.info("fetchOpenseaCreator")
+        val creator = Address.apply(tokenId.value.toByteArray().copyOfRange(0, 20))
+        itemCreatorRepository.save(ItemCreator(itemId, creator))
```
Comment (evgenynacu) [inline nft/core/src/main/kotlin/com/rarible/protocol/nft/core/service/item/ItemCreatorService.kt:70]: this way it won't work. when you do this, nothing will be changed. you have to subscribe to Mono

File: nft/core/src/main/kotlin/com/rarible/protocol/nft/core/service/item/ItemCreatorService.kt [hunk 3]
```diff
@@ -59,8 +64,14 @@ class ItemCreatorService(
             }
     }
 
+    private fun fetchOpenseaCreator(tokenId: EthUInt256, itemId: ItemId): Mono<Address> {
+        logger.info("fetchOpenseaCreator")
+        val creator = Address.apply(tokenId.value.toByteArray().copyOfRange(0, 20))
```
Comment (evgenynacu) [inline nft/core/src/main/kotlin/com/rarible/protocol/nft/core/service/item/ItemCreatorService.kt:69]: That's not a correct way to transform BigInteger to byte array. use Uint256Type

**Review by evgenynacu — CHANGES_REQUESTED** [review]

File: nft/core/src/main/kotlin/com/rarible/protocol/nft/core/service/item/ItemCreatorService.kt [hunk 4]
```diff
@@ -8,19 +8,25 @@ import com.rarible.protocol.nft.core.model.KNOWN_ORIGIN_TOKEN
...
 import scalether.domain.Address
 import scalether.transaction.MonoTransactionSender
 import java.math.BigInteger
 
 @Service
 class ItemCreatorService(
     sender: MonoTransactionSender,
-    private val itemCreatorRepository: ItemCreatorRepository
+    private val itemCreatorRepository: ItemCreatorRepository,
+    @Value("\${common.openseaLazyMintAddress}") private val openseaLazyMint: String
```
Comment (sanufriev) [inline nft/core/src/main/kotlin/com/rarible/protocol/nft/core/service/item/ItemCreatorService.kt:24]: We don't use @Value in code, please use NftIndexerProperties injection

File: nft/api/src/main/resources/config/application.yml (hunk 1 above)
Comment (sanufriev) [inline nft/api/src/main/resources/config/application.yml:62]: It better to define it in prod profile config.
And what about dev/e2e configs?

**Pull request merged**